inference_segmentation.py

```diff
@@ -325,7 +325,7 @@ def main(params: Union[DictConfig, dict]) -> None:
     )
 
     # Dataset params
-    bands_requested = get_key_def('bands', params['dataset'], default=("red", "blue", "green"), expected_type=Sequence)
+    bands_requested = get_key_def('bands', params['dataset'], default=[1, 2, 3], expected_type=Sequence)
```

Comment:
Can we put information on what is 1, 2, 3 ....

Reply:
https://github.com/NRCan/geo-deep-learning/pull/343#discussion_r989440843